src/swarm/neo/AddrPort.d

```diff
@@ -121,12 +122,51 @@ public struct AddrPort
         return false;
     }
 
+    /***************************************************************************
+
+        Converts the address of this instance to a string in the the well-known
+        IPv4 dotted-decimal notation such as "192.168.2.111". The result is
+        written to `dst` as a NUL-terminated string.
+        `dst.length` needs to be at least `INET_ADDRSTRLEN` (from
+        `core.sys.posix.netinet.in_`).
+
+        To allocate a new buffer for the address, use
+        ---
+            AddrPort ap;
+            mstring str = ap.getAddress(new char[INET_ADDRSTRLEN]);
+        ---
+
+        Params:
+            dst = destination string; the minimum required length is
+                `INET_ADDRSTRLEN`
+
+        Returns:
+            a slice referencing the result in `dst` (excluding the terminating
+            NUL byte).
+
+    ***************************************************************************/
+
+    public mstring getAddress ( mstring dst )
+    {
+        assert(dst.length >= INET_ADDRSTRLEN,
```

Comment:
To avoid reintroducing assertions into swarm (when they've all just been removed in v5.x.x), we should try adding a `verify` function in swarm, to be replaced with the one in ocean in v5.x.x.

Reply:
Issue: #286